src/applications/vaos/services/appointment/index.js

```diff
@@ -226,15 +226,7 @@ export function getVARFacilityId(appointment) {
       return appointment.legacyVAR.apiData.facilityId;
     }
 
-    const id = appointment.participant?.[0]?.actor?.reference
-      ?.split('/')?.[1]
-      ?.split('_')?.[0];
-
-    if (id) {
-      return id;
-    }
-
-    return null;
+    return appointment.location?.vistaId;
```

Comment:
I think you could also remove the `if (appointment.vaos?.isVideo) {` condition, since video appointments should have the right id in `vistaId` as well.

Reply:
Done.